app/views/shared/_owned_table.html.erb

```diff
@@ -38,7 +38,9 @@
         </td>
         <td>
           <% if child.class == Upload %>
-            <%= link_to "Delete", upload_path(child), method: :delete, class: "btn"%>
+            <%= link_to "Delete", upload_path(child), method: :delete, class: "btn btn-upload-delete"%>
```

Comment:
@cmacaulay This shouldn't matter much for delete since we should be able to delete folders too.

I think a helper method would be best when we need to perform logic in our view. For example, with the share button, we could create a `share_link` tag and all that in the view instead of the if..link_to combo.